README.md

```diff
@@ -46,6 +46,12 @@ It consists of various modules that aids penetration testing operations:
 	sudo pip install -r requirements.txt
 	./rsf.py
 
+# Running on Docker
+
+Set up docker [for your platform](https://docs.docker.com/)
+
+> ./run_docker.sh
```

Comment:
`run_docker.sh` script is missing in this PR - you could add single parametrized shell script for building and running Docker container.